src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -147,6 +151,12 @@ public ApplicationModel.IResourceBuilder<T> CreateResourceBuilder<T>(T resource)
             where T : ApplicationModel.IResource { throw null; }
     }
 
+    public static partial class DistributedApplicationBuilderExtensions
+    {
+        public static ApplicationModel.IResourceBuilder<T> CreateResourceBuilder<T>(this IDistributedApplicationBuilder builder, string name)
```

Comment:
@ReubenBond we have https://github.com/dotnet/aspire/blob/b45d73e45a0350cbe1f1266228bcb0597a8a19fa/src/Aspire.Hosting/DistributedApplicationBuilder.cs#L448-L454. I guess this needs to be an extension method?